PatchUtils.cpp

```diff
@@ -718,45 +709,28 @@ namespace PatchUtils
 
                 // On the first callback, we must recalculate the substituted bytes for the hook strings
                 // of the unmodified ROM. This must occur strictly before the new patch list chunk is created.
+                // We don't need to redo this part if rom size expanding happens
                 if(firstCallback)
                 {
+                    // just remove all the old patches from the ROM
+                    for (struct PatchEntryItem &patch : removePatches)
```

Comment:
Does removePatches at this place always contain all the current patches in the ROM?

Reply:
yes